bhagavad_gita_api/data/insert/commentaries.py

```diff
@@ -19,12 +19,12 @@
 for i in track(data, description="Loading commentary"):
     li.append(
         GitaCommentary(
-            description=i.get("description"),
-            author_name=i.get("authorName"),
-            language=i.get("lang"),
-            verse_id=i.get("verseNumber"),
-            author_id=i.get("author_id"),
-            language_id=i.get("language_id"),
+            description=i["description"],
+            author_name=i["authorName"],
+            language=i["lang"],
+            verse_id=i["verseNumber"],
```

Comment:
We removed the empty verse_number and kept verse_id
because verse_id is used everywhere to keep a track of relations and changing that had potential to invite unwanted bugs
so , the issue of empty column is solved either way
thanks
Hare Krishna